### PR TITLE
:sparkles: Adding SHA sums and renaming location to type

### PR DESCRIPTION
**Pull request by shawn-hurley**
fixes: #73

### CODE REVIEW

File: provider/java/dependency.go [hunk 1]
```diff
@@ -37,8 +38,25 @@ func (p *javaProvider) GetDependencies() ([]dependency.Dep, error) {
...
+	}
+
+	// check errors
+	return string(outb.String())
+}
+
 func (p *javaProvider) GetDependenciesLinkedList() (map[dependency.Dep][]dependency.Dep, error) {
 
+	localRepoPath := p.getLocalRepoPath()
+	fmt.Printf("here: %v", localRepoPath)
```
Comment (pranavgaikwad) [inline provider/java/dependency.go:58]: remove fmt.Printf?

**Review by pranavgaikwad — CHANGES_REQUESTED** [review]

File: provider/java/dependency.go (hunk 1 above)
Comment (shawn-hurley) [reply provider/java/dependency.go:58]: good catch

File: provider/java/dependency.go [hunk 2]
```diff
@@ -37,8 +38,24 @@ func (p *javaProvider) GetDependencies() ([]dependency.Dep, error) {
 	return deps, err
 }
 
+func (p *javaProvider) getLocalRepoPath() string {
+	cmd := exec.Command("mvn", "help:evaluate", "-Dexpression=settings.localRepository", "-q", "-DforceStdout")
+	var outb bytes.Buffer
+	cmd.Stdout = &outb
+	err := cmd.Run()
+	if err != nil {
+		fmt.Printf("here: %v", err)
```
Comment (pranavgaikwad) [inline provider/java/dependency.go:47]: ~this guy too :)~

Actually I think we need this error reported right?

**Review by pranavgaikwad — APPROVED** [review]

File: provider/java/dependency.go (hunk 2 above)
Comment (shawn-hurley) [reply provider/java/dependency.go:47]: yeah, it probably should be a log but this is a good first issue we have these all over the place

**Pull request merged**